src/cache/containers/crc_set_container.cpp

```diff
@@ -0,0 +1,93 @@
+// Rafael Sá Menezes - 04/2020
+
+#include "crc_set_container.h"
```

Comment:
Check whether we're adding the includes in the correct way (see other examples in ESBMC).